api/tests/data/mocked_data/data/processor/config/areas/ALL.json

```diff
@@ -1 +1 @@
-{"global_area_all": {"ViolationThreshold": -1, "NotifyEveryMinutes": -1, "Emails": "", "EnableSlackNotifications": "false", "DailyReport": "false", "DailyReportTime": "N/A", "OccupancyThreshold": -1, "Id": "ALL", "Name": "ALL"}}
\ No newline at end of file
+{"global_area_all": {"ViolationThreshold": 0, "NotifyEveryMinutes": 0, "Emails": "", "EnableSlackNotifications": "False", "DailyReport": "False", "DailyReportTime": "06:00", "OccupancyThreshold": 0, "Id": "ALL", "Name": "ALL"}, "occupancy_rules": []}
```

Comment:
Maybe the booleans should be booleans and not strings?

Reply:
Of course, I change the process of creating and modifying the ALL.json file and now the boolean values are stored.